### DISCUSSION TITLE
How can I use next-auth for non-nextjs project (pure reactjs) ?

### DISCUSSION THREAD

**Discussion by fangwohung** [nextauthjs/next-auth#3394] [Help] [answered]
Is there something like next-auth for client side only ?

**Comment by fangwohung** [comment, marked as answer]
I have solved this by myself thanks to next-auth rest api

**Reply by balazsorban44** [reply]
Nice! Would love to see if you have a working example! Feel free to post it in #2294! :pray:

**Reply by rkreddy99** [reply]
hey @balazsorban44 were you able to implement this
